### ISSUE TITLE
DinoV2-vs-ResnetClassification

### ISSUE THREAD

**Issue by SameerShanbhogue** [purnasai/Dino_V2#5]
Hi @purnasai 

In DinoV2-vs-ResnetClassification notebook. In Resnet model1 training step - optimizer used is initialized with model.parameters(), which is of DinoV2 model. Shouldn't we use a dedicated optimizer for model1 training like optimizer1 = optim.Adam(model1.parameters(), lr=0.000001). The Resnet accuracy improves to approx 90% using this...

BR
